### ISSUE TITLE
Exception: pyqtwebengine distribution was not found

### ISSUE THREAD

**Issue by raffaem** [invl/pip-autoremove#35]
I am running pip-autoremove on Windows 10 and Anaconda 3, python 3.8

pip-autoremove does not work, the following exception is issued:

```
$ pip-autoremove tensorflow
Traceback (most recent call last):
  File "c:\users\raffaele\anaconda3\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\raffaele\anaconda3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Raffaele\anaconda3\Scripts\pip-autoremove.exe\__main__.py", line 7, in <module>
  File "c:\users\raffaele\anaconda3\lib\site-packages\pip_autoremove.py", line 106, in main
    autoremove(args, yes=opts.yes)
  File "c:\users\raffaele\anaconda3\lib\site-packages\pip_autoremove.py", line 21, in autoremove
    dead = list_dead(names)
  File "c:\users\raffaele\anaconda3\lib\site-packages\pip_autoremove.py", line 29, in list_dead
    graph = get_graph()
  File "c:\users\raffaele\anaconda3\lib\site-packages\pip_autoremove.py", line 89, in get_graph
    for req in requires(dist):
  File "c:\users\raffaele\anaconda3\lib\site-packages\pkg_resources\__init__.py", line 465, in get_distribution
    dist = get_provider(dist)
  File "c:\users\raffaele\anaconda3\lib\site-packages\pkg_resources\__init__.py", line 341, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "c:\users\raffaele\anaconda3\lib\site-packages\pkg_resources\__init__.py", line 884, in require
    needed = self.resolve(parse_requirements(requirements))
  File "c:\users\raffaele\anaconda3\lib\site-packages\pkg_resources\__init__.py", line 770, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'pyqtwebengine<5.13; python_version >= "3"' distribution was not found and is required by the application
```

**Comment by BPylypenko** [comment]
after that:
> pkg_resources.DistributionNotFound: The '**helpdev>=0.6.10**' distribution was not found and is required by the application

and


> pkg_resources.DistributionNotFound: The '**ptyprocess>=0.5**' distribution was not found and is required by the application

**Comment by tresni** [comment]
See #16  and [my branch](https://github.com/tresni/pip-autoremove)

**Comment by tresni** [comment]
Should be addressed by the merge of #42 and the new release, please close if that resolves your issue.
